### ISSUE TITLE
First!

### ISSUE THREAD

**Issue by bazbt3** [bazbt3/topics#2]
And, after all the effort expended in creating **topics**, Baz had nothing to talk about, which was faintly embarrassing.

Incidentally, speaking of *embararassing*, in addition to mis-spelling it in my first Gitter comment I just created an issue at the *Octodroid repo*, thinking I was here.

**Comment by vielmetti** [comment]
First comment.

Yes, keeping track of where you are when you go to make a comment or start an issue is confusing. It all looks the same! What we need are some themes to bling up the scene.

**Comment by bazbt3** [comment]
Indeed. Issues simply aren't designed for *chatting*, it's perhaps more of a linear progression through fixing stuff?

I believe it's why Gitter was created, it'll likely promote a more chatty atmosphere.  Not ready to give up yet though. <grins insanely>

Thanks for the comment. :)

**Comment by matigo** [comment]
Cool. How soon until you get some VC funding and a ping pong table?

**Comment by bazbt3** [comment]
@matigo My daughters have seemingly-limitless reserves of cash, but my vision for a games room goes only as far as Ludo I'm afraid.
